changelog.d/9280.misc

```diff
@@ -0,0 +1 @@
+Ripped out an option from `homeserver.yaml`.
```

Comment:
```suggestion
Removed support for the deprecated `tls_fingerprints` configuration setting.
```

You can add `Contributed by <your name>.` to the end if you would like!

This should also be named `changelog.d/9280.removal`, I think.

Reply:
Thanks for the review and sure adding name won't hurt.